src/common/constants.ts

```diff
@@ -69,12 +69,10 @@ export const oauth_url = `https://oauth.${getDomainUrl()}`
 export const deriv_app_login = `${oauth_url}/oauth2/authorize?app_id=${deriv_app_id}`
 export const whatsapp_url = `https://wa.me/35699578341`
 
-export const application_id = 'f0aef779-d9ec-4517-807e-a84c683c4265'
```

Comment:
Is this one and the live chat ones a few lines bellow all public tokens?

Reply:
yes @kaveh-deriv  and actually we are not using these public tokens , we are taking them from env variable so we decided to remove